components/frontend/package.json

```diff
@@ -24,14 +24,16 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "cov": "react-scripts test --coverage --watchAll --transformIgnorePatterns 'node_modules/(?!history)/'",
     "test": "react-scripts test --maxWorkers 2 --transformIgnorePatterns \"node_modules/(?!history)/\"",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
   "jest": {
-    "resetMocks": false
+    "resetMocks": false,
+    "coveragePathIgnorePatterns": [
+        "<rootDir>/node_modules/",
+        "<rootDir>/src/index.js"
```

Comment:
Why do we ignore this, is it a generated file?

Reply:
I don't see how it can be covered from the unit tests... ☹️